mods/ra/rules/defaults.yaml

```diff
@@ -831,6 +831,7 @@
 		CrushClasses: mine
 		DetonateClasses: mine
 		AvoidFriendly: false
+		BlockFriendly: true
```

Comment:
You won't need to explicitely define this, as `true` is the default value (you set [here](https://github.com/OpenRA/OpenRA/pull/12337/files#diff-4fa252f4c4f2398df3d9099b8b8ee37bR22)).